src/rabbitmq.js

```diff
@@ -671,7 +665,32 @@ class RabbitMQ {
         this.subscribed = this.subscribed.add(subscribedKey)
       })
   }
-
+  /**
+   * Create payload object with `jobBuffer` and `jobMeta` props
+   * @private
+   * @param {Object} content Content to send.
+   * @return {Object} payload with `jobBuffer` and `jobMeta` props
+   */
+  static buildPayload (content: Object) {
```

Comment:
hmm actually, this function is doing 2 independent things, building payload and building data. split out into 2 nice small functions? `getPayload` `getHeaders` ?